src/sentry/replays/usecases/ingest/slow_click.py

```diff
@@ -0,0 +1,69 @@
+import datetime
+from typing import Any, Dict
+
+from sentry import options
+from sentry.issues.grouptype import ReplaySlowClickType
+from sentry.replays.usecases.ingest.events import SentryEvent
+from sentry.replays.usecases.issue import new_issue_occurrence
+
+
+def report_slow_click_issue(project_id: int, replay_id: str, event: SentryEvent) -> bool:
+    # Only report slow clicks if the option is enabled.
+    if not options.get("replay.issues.slow_click"):
+        return False
+
+    payload = event["data"]["payload"]
+
+    # Only timeout reasons on <a> and <button> tags are accepted.
+    if payload["data"]["node"]["tagName"] not in ("a", "button"):
+        return False
+    elif payload["data"]["endReason"] != "timeout":
+        return False
+
+    # Seconds since epoch is UTC.
+    timestamp = datetime.datetime.fromtimestamp(payload["timestamp"])
+    timestamp = timestamp.replace(tzinfo=datetime.timezone.utc)
+
+    _report_slow_click_issue(
+        environment="prod",
+        fingerprint=payload["message"],
+        project_id=project_id,
+        release="",
+        subtitle=payload["message"],
+        timestamp=timestamp,
+        extra_event_data={
+            "level": "info",
+            "tags": {"replay_id": replay_id},
```

Comment:
if the issue platform events support contexts, you can add the replay id to the event with a context:

```
contexts: {
 "replay": {
     "replay_id": replay_id
  }
}
```

if this platform can't do contexts, it's annoying but the tag name we use on events for replay_ids is `replayId`, not `replay_id`